src/translations/screens/TripSearch.ts

```diff
@@ -117,6 +117,10 @@ const TripSearchTexts = {
         'Prøv å justere på sted eller tidspunkt.',
         'Try adjusting your time or location input.',
       ),
+      genericHintWithFilters: _(
+        'Prøv å justere på sted, filtere eller tidspunkt.',
```

Comment:
I think this is more correct grammar ([filter in dictionary](https://ordbokene.no/bm/search?q=filter&scope=ei))

```suggestion
        'Prøv å justere sted, filter eller tidspunkt.',
```